tests/test_reverse_string.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase, main
+from task_2.reverse_string import reverse_string, NonStringInputError
+
+
+class TestReverseString(TestCase):
+    def test_reverse_string(self):
```

Comment:
Test cases looks good, I want you to adopt a nice style practice of writing unit tests called Given - When - Then
WIth that, you basically divide your test into 3 blocks that are separated by the comment
```
def test..():
   # Given
    {happens all the test setup, declaration of variables etc}
   # When
    {Actual code execution that you want to check happens}
   # Then
    {All the asserts happen}
```
Example:
```
def test_reverse_string(self):
    # Given
    cases = [
            ("abcd efgh", "dcba hgfe"),
            ("a1bcd efg!h", "d1cba hgf!e"),
            ("", ""),
        ]
     # When
     for input_text, expected_result in cases:
            with self.subTest(input_text=input_text):
                # Then
                self.assertEqual(reverse_string(input_text), expected_result)
```
It gives a clear division when the things happen. Please adopt it in your two tests and use in further tasks

Reply:
I will use this template too, thanks!